### PR TITLE
Rotate CW or CCW

### PR DESCRIPTION
**Pull request by marksands**
The rotation animation is great! But we modified it so that we can rotate clockwise or counterclockwise.

Is this something that you think is desirable?

### CODE REVIEW

File: IBAnimatable/Animatable.swift [hunk 1]
```diff
@@ -548,11 +550,11 @@ public extension Animatable where Self: UIView {
       }, completion: completion)
   }
 
-  public func rotate(completion: AnimatableCompletion? = nil) {
+  public func rotate(clockwise clockwise: Bool, completion: AnimatableCompletion? = nil) {
```
Comment (JakeLin) [inline IBAnimatable/Animatable.swift]: I prefer to have a default value to `true`, what do you think?

File: IBAnimatable/Animatable.swift [hunk 2]
```diff
@@ -174,8 +174,10 @@ public extension Animatable where Self: UIView {
       wobble(completion)
     case .Swing:
       swing(completion)
-    case .Rotate:
-      rotate(completion)
+    case .RotateCW:
```
Comment (JakeLin) [inline IBAnimatable/Animatable.swift]: Can we keep `Rotate` as `RotateCW` and add `RotateCCW` for counterclockwise?

**Comment by JakeLin** [comment]
@marksands Thanks for your PR, it is very useful to have anticlockwise/counterclockwise for `Rotate`. :+1:  Could you please have a look at my small comments and?

File: IBAnimatable/Animatable.swift (hunk 1 above)
Comment (marksands) [inline IBAnimatable/Animatable.swift]: @JakeLin Yes, I agree

File: IBAnimatable/Animatable.swift (hunk 2 above)
Comment (marksands) [inline IBAnimatable/Animatable.swift]: :+1:

**Comment by marksands** [comment]
@JakeLin Look over the new changes and let me know what you think! :+1:

**Comment by JakeLin** [comment]
@marksands perfect, thanks.👍

**Pull request merged**